psydac/feec/pull_push.py

```diff
@@ -336,9 +336,22 @@ def fun(xi1, xi2, xi3):
 
     return fun
 
+#==============================================================================
+# PUSH-FORWARD operators:
+#   These push-forward operators take logical coordinates,
+#   so they just transform the values of the field.
+#   For H1 push-forward there is no transform, so no mapping is involved.
+#==============================================================================
+
 #==============================================================================
 # 1D PUSH-FORWARD
 #==============================================================================
+def push_1d_h1(func, xi1):
+
+    value     = func(xi1)
+
+    return value
```

Comment:
Differently than in the other push-forwards, here there is no need to create a temporary variable called `value`. So we can just write
```python
def push_1d_h1(func, xi1):
    return func(xi1)
```
and similarly for the 2D and 3D cases.